docs/tools/vdb_table/data/rockset.json

```diff
@@ -14,91 +14,91 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "SQL"
```

Comment:
this should be the language the underlying vector search index / engine is implemented in

Reply:
updated